connectors/aws-cert/src/main/java/com/vmware/connectors/aws/cert/AwsCertController.java

```diff
@@ -259,7 +259,7 @@ private Card makeCard(
         CardAction.Builder dismissAction = CardAction.Builder
                 .dismissAction()
                 .setLabel(cardTextAccessor.getActionLabel("dismiss"))
-                .setCompletedLabel(cardTextAccessor.getActionCompletedLabel("dismiss"));
+                .setCompletedLabel(cardTextAccessor.getActionLabel("dismiss"));
```

Comment:
You shouldn't even need this line (you can let it default to "Completed"), but it's not worth holding this PR up.

Reply:
If ```completed_label``` is missing the card response, then the test cases will fail since it is a mandatory field in ```hero_card_schema.json```.

Reply:
https://github.com/vmware/connectors-workspace-one/blob/5b545876c0e7484fe8a799eb63384900ed4e4eeb/common/core/src/main/java/com/vmware/connectors/common/payloads/response/CardAction.java#L73